sonar-ws/src/main/java/org/sonarqube/ws/client/qualityprofile/QualityProfilesService.java

```diff
@@ -19,34 +19,43 @@
  */
 package org.sonarqube.ws.client.qualityprofile;
 
+import org.sonar.api.server.ws.WebService;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.sonar.api.server.ws.WebService'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)